test/OpenTelemetry.Contrib.Instrumentation.Runtime.Tests/RuntimeMetricsTests.cs

```diff
@@ -54,5 +53,55 @@ public void RuntimeMetricsAreCaptured()
             var metric1 = exportedItems[0];
             Assert.StartsWith(MetricPrefix, metric1.Name);
         }
+
+        [Fact]
+        public void CpuUsageMetricAreCaptured()
+        {
+            var exportedItems = new List<Metric>();
+
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                 .AddRuntimeMetrics(options =>
+                 {
+                     options.CpuEnabled = true;
+                 })
+                 .AddInMemoryExporter(exportedItems)
+                .Build();
+
+            meterProvider.ForceFlush(MaxTimeToAllowForFlush);
```

Comment:
right after this, we can validate that the value is 0. (1st time we return 0 as cpu usage, right)